pkg/cli/settings.go

```diff
@@ -53,3 +60,51 @@ var cmdSettings = &cobra.Command{
 		printSettings(clusterSettings.TransientSettings, "transient settings")
 	},
 }
+
+var cmdSettingsReloadSecure = &cobra.Command{
+	Use:   "reload",
+	Short: "Reload the secure settings.",
+	Long:  `This command calls the reload secure settings API on all nodes.`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		v := getClient()
+
+		password, err := cmd.Flags().GetString("keystore_password")
+		if err != nil {
+			fmt.Printf("Could not retrieve required argument: keystore_password. Error: %s\n", err)
+			os.Exit(1)
+		}
+
+		var reloadResponse vulcanizer.ReloadSecureSettingsResponse
+		var reloadError error
+
+		if password == "" {
```

Comment:
Not really important but would it be more clear if we have just a single method `ReloadSecureSettings(pass string)` and then we decide in the method how to proceed?

Reply:
I was debating that, but thinking about the library use-case I didn't like the idea of having to call `ReloadSecureSettings("")` if I didn't have a password.